odoo/addons/base/models/ir_ui_view.py

```diff
@@ -194,10 +194,11 @@ def get_view_arch_from_file(filename, xmlid):
 xpath_utils = etree.FunctionNamespace(None)
 xpath_utils['hasclass'] = _hasclass
 
+TRANSLATED_ATTRS = [attr for attr in TRANSLATED_ATTRS if attr != 'src']
```

Comment:
Watch out, you must not modify the original `TRANSLATED_ATTRS`, which is what you'd be doing with the inplace set difference. And to better indicate that the modified set is only used temporarily for building the regex and not to replace the original set, it'd be better to lowercase it (and to add a comment to explain why, indeed)

Reply:
I wasn't aware that `a -= b` is not semantically equivalent to `a = a - b` but acts in place in Python :/ live and learn. Good catch!

@pgu-odoo I would still agree with odony's original suggestion of using a different name (or using the same but in lower case) simply because someone who is familiar with `TRANSLATED_ATTRS` may not expect it to not be the same thing in this file as the unmodified imported version.